serialization/README.md

```diff
@@ -2,13 +2,36 @@
 
 ## Serialization info
 
-_TBD_
+This directory contains the specifications and examples for all supported SPDX formats.
+
+There is a specification markdown file for each supported format describing the serialization.
+
+A collection of elements can be serialized in multiple formats.
+
+Within the model, we have an "XCollection" which represents the common properties of the collection of elements across all data formats.
+
+We can refer to the actual serialized bytes as and Artifact in the model.
+
+A Relationship of type "XCollectionOf" can be used to relate an "XCollection" to one more serialized for of the "XCollection".
+
+Within serialized data, you may need to refer to properties in the "XCollection" - such as "where to start" in a collection.
+
+In that case, the "XCollection" will be in the serialized data with ONLY the properties not supported by the serialization format.
+
+There is at most one "XCollection" in the serialization.
```

Comment:
Suggest slight tweak:

"There is at most one "XCollection" element defined in any given instance of serialization."

There can be any number of references via id but only a single XCollection defined within a given instance of serialization.